### PR TITLE
Changed order of Best Practices page and linked it from other relevant pages

### PR DESCRIPTION
**Pull request by leanneeliatra**
### Description
The best practices page was located in the middle of the 'Security in OpenSearch' documentation. It is better placed at the top and therefore the location has been changed.

Also, this page is relevant for other pages, it has been linked on other pages where this link is appropriate, namely:
https://opensearch.org/docs/latest/security/
https://opensearch.org/docs/latest/getting-started/security/

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/7376

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _getting-started/security.md [hunk 1]
```diff
@@ -190,3 +190,9 @@ To upload a configuration, the following command can be used with the admin cert
 ./plugins/opensearch-security/tools/securityadmin.sh -cd "config/opensearch-security" -icl -key "../kirk-key.pem" -cert "../kirk.pem" -cacert "../root-ca.pem" -nhnv
 ```
 {% include copy.html %}
+
+## Next steps
+
+Our best practices guide covers 10 things to consider when getting started with OpenSearch security. [Best practices for OpenSearch security]({{site.url}}{{site.baseurl}}/security/configuration/best-practices/).
```
Comment (Naarcha-AWS) [inline _getting-started/security.md:196]:
```suggestion
The [Best practices for OpenSearch security]({{site.url}}{{site.baseurl}}/security/configuration/best-practices/)  guide covers 10 things to consider when getting started with OpenSearch security.
```

File: _getting-started/security.md [hunk 2]
```diff
@@ -190,3 +190,9 @@ To upload a configuration, the following command can be used with the admin cert
...
+
+You can also check out the configuration overview in the [Security configuration]({{site.url}}{{site.baseurl}}/security/configuration/index/) section, which provides the basic steps for setting up security in your OpenSearch implementation and includes links to information about customizing security for your business needs.
```
Comment (Naarcha-AWS) [inline _getting-started/security.md:198]:
```suggestion
The [Security configuration]({{site.url}}{{site.baseurl}}/security/configuration/index/) overview provides the basic steps for setting up security in your OpenSearch implementation and includes links to information about customizing security for your business needs. 
```

**Review by Naarcha-AWS — APPROVED** [review]
Two small changes, otherwise LGTM.

**Review by hdhalter — APPROVED** [review]
LGTM with Nate's changes. Thanks!

File: _security/index.md [hunk 3]
```diff
@@ -75,5 +75,8 @@ To learn more, see [Cross-cluster search]({{site.url}}{{site.baseurl}}/security/
 
 ## Next steps
 
-To get started, see the configuration overview in the [Security configuration]({{site.url}}{{site.baseurl}}/security/configuration/index/) section, which provides the basic steps for setting up security in your OpenSearch implementation and includes links to information about customizing security for your business needs.
+Our best practices guide covers 10 things to consider when getting started with OpenSearch security. [Best practices for OpenSearch security]({{site.url}}{{site.baseurl}}/security/configuration/best-practices/).
```
Comment (Naarcha-AWS) [inline _security/index.md:78]:
```suggestion
The [Best practices for OpenSearch security]({{site.url}}{{site.baseurl}}/security/configuration/best-practices/)  guide covers 10 things to consider when getting started with OpenSearch security.
```

File: _security/index.md [hunk 4]
```diff
@@ -75,5 +75,8 @@ To learn more, see [Cross-cluster search]({{site.url}}{{site.baseurl}}/security/
...
+
+You can also check out the configuration overview in the [Security configuration]({{site.url}}{{site.baseurl}}/security/configuration/index/) section, which provides the basic steps for setting up security in your OpenSearch implementation and includes links to information about customizing security for your business needs.
```
Comment (Naarcha-AWS) [inline _security/index.md:80]:
```suggestion
The [Security configuration]({{site.url}}{{site.baseurl}}/security/configuration/index/) overview provides the basic steps for setting up security in your OpenSearch implementation and includes links to information about customizing security for your business needs. 
```

**Pull request merged**